### PR TITLE
2.1.7 cms url

### PR DESCRIPTION
**Pull request by thescubageek**
* Refactors Formatter and CMS url methods into separate concern, which fixes bug in GardenWebThemeUpdater.
* Patches Naked Theme

### CODE REVIEW

File: app/models/widget.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Widget < ActiveRecord::Base
   include WidgetSettings
   include UpdateNavSettings
   include LiquidParameters
+  include CmsUrlFormatter
```
Comment (thescubageek) [inline app/models/widget.rb]: This is necessary for future work on Click2Edit

**Comment by thescubageek** [comment]
@crissmancd @jcstringer Quick review on this one? It gets custom layouts to work on Naked plus refactors some ugly code

File: app/models/cms_url_formatter.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module CmsUrlFormatter
```
Comment (jcstringer) [inline app/models/cms_url_formatter.rb]: I think you prob meant to put this file in the concerns folder instead of models

**Comment by jcstringer** [comment]
Nice work DRY'ing up all that code!

File: app/models/cms_url_formatter.rb (hunk 2 above)
Comment (thescubageek) [inline app/models/cms_url_formatter.rb]: D'oh, of course I did. I had this set up correctly on Click2Edit branch but needed it in earlier to make the theme fix work, moved it over all wrong...

**Pull request merged**